NetworkingSampleApp/NetworkingSampleApp/ContentView.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  ContentView.swift
+//  NetworkingSampleApp
+//
+//  Created by Matej Molnár on 28.01.2023.
+//
+
+import SwiftUI
+
+enum Sample: String, Hashable, CaseIterable {
```

Comment:
Ok we have sample app, but this is too much I would say :-D enum Sample :-D

What does it represent actually?

Reply:
Got it 😄 Do you think renaming to Example would be better?